libraries/AP_BoardConfig/AP_BoardConfig.cpp

```diff
@@ -116,7 +116,7 @@ const AP_Param::GroupInfo AP_BoardConfig::var_info[] = {
     // @Param: SERIAL_NUM
     // @DisplayName: User-defined serial number
     // @Description: User-defined serial number of this vehicle, it can be any arbitrary number you want and has no effect on the autopilot
-    // @Range: -32767 32768
+    // @Range: -32767 32767
```

Comment:
Not very important but we should put -32768 there, which is what was probably the initial intention